client/lib/post-normalizer/utils/max-width-photonish-url.js

```diff
@@ -13,9 +13,9 @@ export function maxWidthPhotonishURL( imageURL, width ) {
 		return imageURL;
 	}
 
-	let parsedURL = {};
+	let urlParts = {};
 	try {
-		parsedURL = url.parse( imageURL, true, true ); // true, true means allow protocol-less hosts and parse the querystring
+		urlParts = getUrlParts( imageURL ); // true, true means allow protocol-less hosts and parse the querystring
```

Comment:
```suggestion
		urlParts = getUrlParts( imageURL );
```

does this still handle protocol-neutral URLs? like `//foo.com/bar.gif`

Reply:
Yes, `getUrlParts` handles all types of URL 👍